data/com.endlessm.GoogleChromeInitialSetup.gschema.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm.. good point. To be honest, this morning I was thinking we can drop the whole gsettings-related logic altogether, as in the end it's irrelevant in most cases: if a user had an image with that feature initially enabled, the feature would likely be disabled after an OSTree upgrade, since the Chrome download would likely win over a OS upgrade.

And in that case, the gsetting would be disabled at the end of the process (and if it was not, which could be due to bugs found recently), that would be a mistake.

So, I'll remove the rest of the GSettings stuff